app/Schedule/View.elm

```diff
@@ -269,6 +274,16 @@ predictionColor minutesLate =
             Color.red
 
 
+arrival : Date -> Train -> Node Msg
+arrival date train =
+    trainMetadata ("arrives at" ++ arrivalTime train)
+
+
+arrivalTime : Train -> String
+arrivalTime train =
+    prettyTime train.scheduledArrival
```

Comment:
http://package.elm-lang.org/packages/elm-lang/core/5.1.1/Basics#<|
http://package.elm-lang.org/packages/elm-lang/core/5.1.1/Basics#<<

```elm
prettyTime : String -> String
scheduledArrival : Train -> String
```

So, the signature for `prettyTime << .scheduledArrival` is `Train -> String`, which lines up with the type for `arrivalTime`. So, instead of explicitly including the argument `train`, which is then immediately applied:

```elm
arrivalTime train =
    (prettyTime << .scheduledArrival) train
```

we can reduce it to:

```elm
arrivalTime =
    prettyTime << .scheduledArrival
```